help/en/html/tools/logixng/reference/chapter5.shtml

```diff
@@ -972,16 +972,43 @@ LogixNG: IQ:AUTO:0001
 
       <h4  id="ExecuteDelayed">Execute delayed</h4>
 
-      <p>Delay the execution of the following action for a specified amount of time. This is the
-      same as the Logix <strong>Delayed Set ...</strong>. The main difference is that this can be
-      used with any action, such as signals. The time delay units are milliseconds, seconds,
-      minutes and hours.</p>
-
-      <p>The default mode is that once a timer starts, subsequent requests will be ignored. If
-      <strong>Restart timer if already started</strong> is selected and a timer is already running,
-      it will be canceled and started again.  The subsequent action will be delayed until the timer
-      finishes.  If <strong>Use individual timers</strong> is selected, each request will be
-      independent.</p>
+      <p>Delay the execution of the following <strong><em>action</em></strong> for a specified amount
+      of time.  The <strong><em>action</em></strong> can be a single action similar to the Logix
+      <strong>Delayed Set ...</strong>, but this can be any item, such as signals.  The
+      <strong><em>action</em></strong> can be a series of actions by using the <strong>Many</strong>
+      action.  It can be a complex action by using an <strong>If-Then-Else</strong> action.</p>
+
+      <p class="noted">Note:  When using If-Then-Else, the <strong>Always execute</strong> option is
+      required.  Any expressions that refer to JMRI table items need to use the <strong>No Listen</strong> option.</p>
```

Comment:
A suggestion:
Any expressions that refer to JMRI table items, like turnouts, sensors and signals, need to use the ...

For me it was not obviously what `JMRI table` items refer to.